index.html

```diff
@@ -324,33 +323,34 @@
       <p>The <dfn>observe()</dfn> method instructs the user agent to
       <dfn>register the observer</dfn> and must run these steps:</p>
       <ol data-link-for="PerformanceObserverInit">
-        <li>Remove all unsupported names from the entryTypes sequence.</li>
-        <li>If the resulting '<a>entryTypes</a>' sequence is an empty sequence, abort
-          these steps. The user agent SHOULD notify developers when the steps are
-          aborted. A console warning listing ignored unsupported names in `entryTypes`
-          might be appropriate, for example.
-        </li>
-        <li>If the list of <a>registered performance observer</a> objects
-        associated with the <a>ECMAScript global environment</a> of the
-        interface object's contains a <a>registered performance observer</a>
-        that is the <a>context object</a>, replace the <a>context object</a>'s
-        `options` with <var>options</var>.
+        <li>Let <var>entry types</var> be <var>options</var> <a>entryTypes</a>
+          sequence.</li>
+        <li>Remove all unsupported types from <var>entry types</var>. The user
+          agent SHOULD notify developers if <var>entry types</var> is modified.
+          For example, a console warning listing removed types might be appropriate.</li>
+        <li>If the resulting <var>entry types</var> sequence is an empty sequence,
+          abort these steps. The user agent SHOULD notify developers when the
+          steps are aborted to notify that registration has been aborted. For
+          example, a console warning might be appropriate.</li>
+        <li>If the <a>list of registered performance observer objects</a> of
+          <a>current global object</a> contains a <a>registered performance
```

Comment:
Why current and not relevant?  What do implementations actually do here?

There's a pretty fundamental conceptual difference between using "current" and "relevant" here, unlike in some cases.  If "relevant" is used, then the observer can only observe things in one global (its "relevant global").  If "current" is used, then the same observer can observe things in multiple globals at once.  Which behavior are you aiming for?  If it's the "current global" behavior, it might be worth a note pointing out the multiple-global subtlety, because it would be easy to miss for both implementors and authors.

If you actually decide you want the "relevant global", I and probably @domenic would like to know why you ended up picking the "current global" here when that wasn't the thing you wanted.  Maybe we can make the HTML spec descriptions for these globals clearer somehow, if they're still confusing.

Either way, given the clear semantic difference above, this needs tests.

Reply:
Thanks for flagging this, the mapping and differences were unclear in my head and staring at "8.1.3.5.3 Current" vs "8.1.3.5.4 Relevant" in HTML spec didn't help. 

Based on the differences you highlighted (which, btw, are a much approachable explainer that could probably make a useful note in the HTML spec, under either section), I believe I have this backwards: we want relevant. I'll update and we can confirm with other implementations.